chapter1/spec/features/search_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "spec_helper"
+
+describe "My search page" do
+  it "has results" do
+    visit("https://google.com")
+    fill_in('q', with: "snappy316")
+    click_on("Google Search")
+    page.has_content?("twitter.com/snappy316")
```

Comment:
This `has_content` method may return true or false, but it is not an assertion. Please ensure every spec has an assertion of some kind. Think: `...must...` or `...wont...`.